public/employers/giant-food.yml

```diff
@@ -0,0 +1,38 @@
+name: Giant Food of Maryland, LLC
+aliases:
+  - Giant-Landover
```

Comment:
Parent of Giant is Ahold Delhaize and has a ticker in Euronext as "AD." I think it's fine keeping it separate since it looks like their other subsidiary, Stop & Shop, is actually handling it better: https://www.boston.com/news/local-news/2020/04/12/provincetown-stop-and-shop-coronavirus-donor

I'm just not sure if you want to include AD as the alias or not.

Reply:
Hm -- I think Ahold Delhaize should actually be the entry. They are regulating processes across a lot of their local stores, according to their news page. https://www.aholddelhaize.com/en/media/latest/media-releases/ahold-delhaize-and-local-brands-deploy-more-than-170-million-on-covid-19-care-to-meet-associate-customer-and-community-needs/

Good catch.